forch/forchestrator.py

```diff
@@ -137,10 +142,9 @@ def initialize(self):
         self._cpn_collector.initialize()
         LOGGER.info('Using peer controller %s', self._get_peer_controller_url())
 
-        if self._calculate_config_files():
+        if self._should_enable_faucetizer:
```

Comment:
Can remove  set_faucetizer_enabled function from other class?

Reply:
In FaucetStateCollector class?